wiki/style.css

```
@@ -0,0 +1,605 @@
+    --accent-hover: #79c0ff;
+    --accent-bg: rgba(56, 139, 253, 0.1);
+    --green: #3fb950;
+    --orange: #d29922;
+    --red: #f85149;
+    --purple: #bc8cff;
+    --sidebar-width: 260px;
+    --toc-width: 220px;
+    --header-height: 56px;
+    --radius: 6px;
+    --shadow: 0 1px 3px rgba(0,0,0,0.3);
+    --font-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)